sql/mysqld.h

```diff
@@ -938,6 +938,15 @@ extern bool max_user_connections_checking;
 extern ulong opt_binlog_dbug_fsync_sleep;
 static const int SERVER_UID_SIZE= 29;
 extern char server_uid[SERVER_UID_SIZE+1];
+extern ulong binlog_error_action;
+
```

Comment:
I referred MySQL’s locations for changes in `sys_vars.cc` and `mysqld.cc`, but `mysqld.h`… I don’t understand how the sections were divided.